framework/test/unit/modules/pos/genesis_block_test_data.ts

```diff
@@ -750,4 +750,51 @@ export const invalidAssets: any[] = [
 		},
 		'Init validators is greater than number of active validators',
 	],
+	[
+		'when genesis block initValidators contains more validators than round length',
```

Comment:
isn't this covered by the test above?

Reply:
Actually it's covered, for some reason I missed it. Let me revert the changes.